src/chart/pie/PieSeries.js

```diff
@@ -45,6 +48,28 @@ var PieSeries = echarts.extendSeriesModel({
         this._defaultLabelLine(option);
     },
 
+    processOptionSeriesValue: function (option) {
+        var data = option.data;
+
+        for(var i = 0; i < data.length; i++){
+            if(data[i].value instanceof Array){
+                //visualMap component fix me
+                // if(data[i].value[0] instanceof Array){
+                //     if(option.visualMap && option.visualMap instanceof Array){
+                //         for(var j = 0; j < option.visualMap.length; j++){
+                //             option.visualMap[j]['value'] = data[i].value[j][1]; //set second number to visualMap...
+                //         }
+                //     }
+                // }
+                data[i].value = data[i].value.reduce(function(a, b) {
```

Comment:
Please do not use `reduce` function. It's not compatitable in IE8

Reply:
Thanks for the reminder， I am try to fix.